Doc/library/typing.rst

```diff
@@ -326,10 +344,21 @@ A user-defined class can be defined as a generic class.
        def log(self, message: str) -> None:
            self.logger.info('%s: %s', self.name, message)
 
-``Generic[T]`` as a base class defines that the class ``LoggedVar`` takes a
+This syntax indicates that the class ``LoggedVar`` takes a
 single type parameter ``T`` . This also makes ``T`` valid as a type within the
 class body.
 
+Generic classes implicitly inherit from :class:`Generic`. For compatibility
+with Python 3.11 and lower, it is also possible to inherit explicitly from
+:class:`Generic`::
+
+   from typing import TypeVar, Generic
+
+   T = TypeVar('T')
+
+   class LoggedVar(Generic[T]):
+       ...
+
 The :class:`Generic` base class defines :meth:`~object.__class_getitem__` so
 that ``LoggedVar[T]`` is valid as a type::
```

Comment:
Since inheritance from `Generic` is now implicit, it might not be immediately obvious that just because `Generic.__class_getitem__` exists, it will also exist on all `Generic` subclasses:

```suggestion
Generic classes have :meth:`~object.__class_getitem__` methods, meaning they
can be parameterised at runtime (e.g. ``LoggedVar[int]`` below)::
```